lib/live_view_native/engine.ex

```diff
@@ -461,6 +461,26 @@ defmodule LiveViewNative.Engine do
     end
   end
 
+  # Slot (for HTML element)
+
+  defp handle_token(
+    {:tag_open, ":" <> _, _attrs, _} = token,
+    %{tags: [{:tag_open, <<first, _::binary>> = parent_name, _, _} | _]} = state
+  ) when not (first in ?A..?Z) and first != ?. do
```

Comment:
I'd add a comment clarifying this when clause and in which cases it doesn't apply

Reply:
Let me know if the comment I added clears it up.

Reply:
lgtm